src/Components/FormComboBox.js

```diff
@@ -0,0 +1,119 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { comboBox } from '../../node_modules/uswds/src/js/components';
+
+import { ValidationState } from '../Models/PatientFieldData';
+import ValidationMessage from './ValidationMessage';
+
+function FormComboBox({ label, property, required, onChange, options, showRequiredHint, size, validationState, value }) {
+  const ref = useRef();
+  const [focused, setFocused] = useState(false);
+  const [customOption, setCustomOption] = useState(null);
+
+  useEffect(() => {
+    const { current } = ref;
+    comboBox.on(current);
+    // manually add event handlers to the custom input added by USWDS
+    const input = current.querySelector('input[type="text"]');
+    input.addEventListener('input', (e) => setCustomOption(e.target.value));
+    input.addEventListener('focus', () => setFocused(true));
+    input.addEventListener('blur', () => setFocused(false));
+    input.value = value;
```

Comment:
Here we do a bunch of custom initialization on the input box added by USWDS for the comboBox. We have to use standard DOM methods in this case...